File: src/taburet/transactions/month_report/reduce.js
function(keys, values, rereduce) {
	var debet = 0
	var kredit = 0
	var before = 0
	
	for each(v in values) {
		if ( v.before === undefined) { 
			debet += v.debet
			kredit += v.kredit
		} else {
			before += v.before
		}
	}
	
	return {debet:Math.round(debet * 100.0) / 100.0,
		kredit:Math.round(kredit * 100.0) / 100.0,
		before:Math.round(before * 100.0) / 100.0,
		after:Math.round((before + debet - kredit) * 100.0) / 100.0
	}
}

File: src/taburet/transactions/month_report/reduce.erl
fun(Keys, Values, ReReduce) ->
	{Debet, Kredit, Before} = lists:foldl(
		fun(V, {OD,OK,OB}) ->
			{[{<<"debet">>, D}, {<<"kredit">>, K}, {<<"before">>, B}|_]} = V,
			{D + OD, K + OK, B + OB}
		end,
		{0, 0, 0},
		Values
	),

	{[
		{debet, erlang:round(Debet*100.0)/100.0},
		{kredit, erlang:round(Kredit*100.0)/100.0},
		{before, erlang:round(Before*100.0)/100.0},
		{'after', erlang:round((Before+Debet-Kredit)*100.0)/100.0}
	]}
end.
